sdk/storage/azure-storage-blob/apilisting.txt

```diff
@@ -0,0 +1,591 @@
+class AccessPolicy(Model, object):
+	permission:permission
+	expiry:expiry
+	start:start
+
+	def __eq__(self, other): ...
+	def __init__(self, permission: permission=None, expiry: expiry=None, start: start=None): ...
+	def __ne__(self, other): ...
+	def __str__(self): ...
+	def as_dict(self, keep_readonly=True, key_transformer: function=<function attribute_transformer at 0x10e6a4510>) -> dict: ...
+	@classmethod
+	def deserialize(cls, data: str, content_type: str=None): ...
+	@classmethod
+	def enable_additional_properties_sending(cls): ...
+	@classmethod
+	def from_dict(cls, data: dict, key_extractors=None, content_type: str=None): ...
+	@classmethod
+	def is_xml_model(cls): ...
+	def serialize(self, keep_readonly: bool=False) -> dict: ...
+	def validate(self) -> list: ...
+
+class AccountPermissions(object):
+	AccountPermissions.ADD:AccountPermissions
+	AccountPermissions.CREATE:AccountPermissions
+	AccountPermissions.DELETE:AccountPermissions
+	AccountPermissions.LIST:AccountPermissions
+	AccountPermissions.PROCESS:AccountPermissions
+	AccountPermissions.READ:AccountPermissions
+	AccountPermissions.UPDATE:AccountPermissions
+	AccountPermissions.WRITE:AccountPermissions
+	read:bool
+	write:bool
+	delete:bool
+	list:bool
+	add:bool
+	create:bool
+	update:bool
+	process:bool
+	_str:str
+
+	def __add__(self, other): ...
+	def __init__(self, read: bool=False, write: bool=False, delete: bool=False, list: bool=False, add: bool=False, create: bool=False, update: bool=False, process: bool=False, _str: str=None, *, AccountPermissions.ADD, AccountPermissions.CREATE, AccountPermissions.DELETE, AccountPermissions.LIST, AccountPermissions.PROCESS, AccountPermissions.READ, AccountPermissions.UPDATE, AccountPermissions.WRITE): ...
```

Comment:
Breaking change: Please remove the `_str` constructor parameter. Internal positional parameters are pure evil.